kong/conf_loader.lua

```diff
@@ -61,6 +61,8 @@ local CONF_INFERENCES = {
   cluster_advertise = {typ = "string"},
   nginx_worker_processes = {typ = "string"},
   upstream_keepalive = {typ = "number"},
+  server_tokens = {typ = "boolean"},
+  latency_tokens = {typ = "boolean"},
 
```

Comment:
What's the conclusion, `server_tokens` vs. `server_headers`? Or something different?